Sources/TuistKit/Services/InitService.swift

```diff
@@ -117,6 +117,7 @@ class InitService {
                 let parsedAttributes = try parseAttributes(
                     name: name,
                     platform: platform,
+                    tuistVersion: Constants.version,
```

Comment:
Using user's current tuist version from `Constants.version` and I don't know if its okay. 
Is there any other way to get which tuist version is running at runtime?

Reply:
We've recently moved the version into the tuist-cloud part of the CLI to ease our release process, and therefore that `Constants.version` is no longer used. What I recommend doing is the following:

```bash
let result = try System.shared.capture([CommandLine.arguments[0], "version"])
```

`CommandLine.arguments[0]` should be the path to Tuist itself, which is usually the first argument of the process being executed, and then you call `version` on it.

Long-term we'll refactor the code so that the implementation is not that meta (tuist calling tuist)